sw/device/lib/dif/dif_spi_device.h

```diff
@@ -909,6 +901,331 @@ OT_WARN_UNUSED_RESULT
 dif_result_t dif_spi_device_get_flash_status_registers(
     dif_spi_device_handle_t *spi, uint32_t *value);
 
+typedef struct dif_spi_device_tpm_caps {
+  /** Hardware revision of the TPM submodule. */
+  uint8_t revision;
+  /**
+   * True if the TPM supports 5 Locality. Otherwise, only 1 Locality is
+   * supported.
+   */
+  bool multi_locality;
+  /**
+   * The maximum transfer size, in bytes, of the data phase of TPM
+   * transactions. This represents the maximum amount of data the FIFO can
+   * accept.
+   */
+  uint32_t max_transfer_size;
```

Comment:
could this be a `uint8_t`?

Reply:
Or a `uint16_t`, just in case it grows to be a larger size (but still reasonable). But we can go to `uint8_t` right now.